test/data/stages/chrony/a.mpp.json

```diff
@@ -0,0 +1,34 @@
+{
+  "pipeline": {
+    "build": {
+      "mpp-import-pipeline": {
+        "path": "../manifests/f32-build.json"
+      },
+      "runner": "org.osbuild.fedora32"
```

Comment:
This and the following commits (correctly) changes from f30 to f32, right? Just so happens not to change the diff?

Reply:
Yeah. I'm amending the commit messages to include the fact that it's also effectively upgrading the tests to fedora-32.

What do you mean by "happens not to change the diff"? `diff.json` don't change in most cases because the stages keep doing the same thing.

Reply:
Yeah, that's what I meant.